doc/api/console.md

```diff
@@ -367,7 +367,7 @@ console.table(undefined);
 ```
 
 ```js
-console.table([{ a: 1, b: 'Y' }, { a: 'Z', b: 2 }]);
+console.table([{ a: 1, b: 'Y' }, { a: 'Z', b: 2 }], ['a', 'b']);
```

Comment:
The parameter `properties` is optional here.
The output is same even if `['a', 'b']` is not passed as `properties`

Reply:
For me:
```js
> console.table([{ a: 1, b: 'Y' }, { a: 'Z', b: 2 }]);
┌─────────┬──────────────────┐
│ (index) │      Values      │
├─────────┼──────────────────┤
│    0    │ { a: 1, b: 'Y' } │
│    1    │ { a: 'Z', b: 2 } │
└─────────┴──────────────────┘
```

Reply:
They are optional, but not with the same output it seems.

Reply:
@vsemozhetbyt You're right. I just verified it with Node.js v10.0.0
My bad, earlier I'd verified it on Google Chrome v66 where the output was same even if `['a', 'b']` is not passed as `properties`